internal/olm/operator/registry/operator_installer.go

```diff
@@ -375,6 +375,8 @@ func (o *OperatorInstaller) getTargetNamespaces(supported sets.String) ([]string
 		return []string{o.cfg.Namespace}, nil
 	case supported.Has(string(v1alpha1.InstallModeTypeSingleNamespace)):
 		return o.InstallMode.TargetNamespaces, nil
+	case supported.Has(string(v1alpha1.InstallModeTypeMultiNamespace)):
+		return o.InstallMode.TargetNamespaces, nil
```

Comment:
Suggestion: Can you please log a warning message here that MultiNamespace is being used, before returning the InstallMode.